content/index.md

```diff
@@ -1,18 +1,22 @@
 ---
 title: OSRA Infrastructure Project
 ---
+Welcome to the Open Robotics Infrastructure Project Site.
```

Comment:
```suggestion
Welcome to the Open Robotics Infrastructure Project.
```

I'm torn about this welcome message. It reminds me a lot of old internet and so I'm fond of it, but it doesn't actually add anything. In any case I think Site is a little redundant.